components/dashboard/src/Login.tsx

```diff
@@ -320,103 +321,39 @@ const LoginContent = ({
     );
 };
 
-const LeftPanel = () => {
-    const { isDark } = useTheme();
-
+const RightProductDescriptionPanel = () => {
     return (
-        <div className="w-full lg:w-1/3 lg:max-w-lg flex flex-col justify-between p-4 lg:p-10 lg:pb-2 min-h-screen">
+        <div className="w-full lg:w-1/3 flex flex-col md:justify-center p-4 lg:p-10 lg:pb-2 md:min-h-screen">
             <div>
-                <div className="p-[1px] bg-gradient-to-b from-white to-[#ECE7E5] dark:from-gray-700 dark:to-gray-600 rounded-2xl justify-center items-center mb-8">
-                    <div className="bg-[#F9F9F9B2] dark:bg-gray-800 w-full p-4 rounded-2xl border border-gray-100 dark:border-gray-700">
-                        <h2 className="bg-white dark:bg-gray-700 inline-flex text-xs font-semibold mb-4 border-[0.5px] shadow border-divider dark:border-gray-600 px-2 py-1 rounded-lg">
-                            <span className="bg-gradient-to-l from-[#FFAE33] to-[#FF8A00] text-transparent bg-clip-text">
-                                Did you know?
-                            </span>
-                        </h2>
-                        <p className="text-pk-content-secondary dark:text-gray-300 mt-1">
-                            We launched a new version of Gitpod in early access.
-                        </p>
-                    </div>
-                </div>
                 <div className="justify-center md:justify-start mb-6 md:mb-8">
-                    <ProductLogo className="h-8 mb-4" />
                     <h2 className="text-2xl font-medium mb-2 dark:text-white inline-flex items-center gap-x-2">
-                        Gitpod Flex
+                        Gitpod Classic
                     </h2>
-                    <p className="text-lg text-gray-600 dark:text-gray-300 mb-2">
-                        Automated, standardized
-                        <br /> development environments.
+                    <p className="text-pk-content-secondary mb-2">
+                        Automated, standardized development environments hosted by us in Gitpod’s infrastructure. Users
+                        who joined before October 1, 2024 on non-Enterprise plans are considered Gitpod Classic users.
```

Comment:
This copy is not very clear, as we still allow signups for Classic. This is a tricky one to describe, though, since basically the only change we made was what https://www.gitpod.io/login pointed to.

Should we instead be a bit more loose in the copy and say something like "If you joined after October 1st, 2024, you are most likely a Gitpod Flex user."?

We sadly lack a good way for the user to be able to tell what they signed up for in the past without trying to log in into both interfaces.

Reply:
> We sadly lack a good way for the user to be able to tell what they signed up for in the past without trying to log in into both interfaces.

We have this on our website now: https://www.gitpod.io/flex-or-classic

which is reachable by `Login` or `Dashboard` buttons in the top navbar.

Reply:
@filiptronicek I think the point to highlight before, not after? With new Login experience between website, flex and classic I hope users can understand where they are.

Reply:
Totally, I think I originally misread this. The "users before Oct 1 are Classic users" statement holds, and we should use that. Sorry for the noise!